frontend/website/pages/docs/tokens.mdx

```diff
@@ -0,0 +1,112 @@
+import Page from '@reason/pages/Docs';
+export default Page({title: "Tokens"});
+
+# Tokens  
+
+Using the Coda CLI (command-line interface) is the primary way to create new token accounts, create new tokens, and mint tokens on the Coda blockchain. 
+
+In this section, you will find out how to use the Coda CLI to create new tokens, create new token accounts, and mint non-default tokens!
+
+## Creating Tokens
+
+With the CLI, you are able to create your own layer 2 tokens! Layer 2 tokens are a new type of token that is introduced to the Coda blockchain. These new types of tokens have a unique identifier that differentiate them from all other tokens in the blockchain. Users with an associated token account can then receive/send this new type of token that is totally separate from the native Coda token itself!  When you create a new token, you are then able to mint more of the token to be used (see below in Minting Tokens). 
```

Comment:
We want to avoid the language "layer 2 tokens" because people may get confused that this is similar to a layer-2 protocol. Can we call this "tokens on Coda"?

Reply:
nit: Make "Minting Tokens" a link

Reply:
Note: please update "layer 2 tokens" elsewhere in the doc as well.